src/shared/components/search-filters/search-filters.component.ts

```diff
@@ -90,7 +107,21 @@ export class SearchFiltersComponent implements OnInit {
     this.selected.page = 1;
     this.selected.metadataFields = this.getMetadataFields();
     this.populateValuesInLocalStorage();
-    this.onApply.emit(this.selected);
+    // Save filters as query parameters for current route
+    let queryParams = {};
```

Comment:
clear duplicated block of code. We should remove this duplication

Reply:
Done